docs/self-hosted-appcircle/install-server/helm-chart/configuration/advanced-configuration.md

```diff
@@ -0,0 +1,615 @@
+---
+title: Advanced Configuration
+description: Learn how to fully configure the Appcircle server Helm chart
+tags: [self-hosted, helm, configuration, kubernetes]
+sidebar_position: 100
+---
+
+import NeedHelp from '@site/docs/\_need-help.mdx';
+
+For advanced configuration options, open the `values.yaml` file with your preferred text editor and modify the settings as needed.
+
+Once you have updated the `values.yaml` file, please proceed to the [Upgrade Appcircle Server](/self-hosted-appcircle/install-server/helm-chart/upgrades.md) section to apply the changes.
+
+
+<!---
+### Custom Testing Distribution Domain
+
+TODO: Fill the post jobs after enabling the custom store domain.
+-->
+
+## Increase the Replica Counts
+
+With the default Helm values, the Appcircle server services being deployed with one replica. If you want to increase this number for high availability, you can do so by updating your `values.yaml` file:
+
+:::caution
+Some keys might already exists in your `values.yaml` file, make sure to update the existing keys instead of adding new ones.
+:::
+
+```yaml
+agentcache:
+  replicaCount: 3
+auth:
+  auth-keycloak:
+    replicas: 3
+apigateway:
+  replicaCount: 2
+appparser:
+  replicaCount: 3
+build:
+  replicaCount: 3
+distribution:
+  distribution-server:
+    replicaCount: 3
+  distribution-testeradmin:
+    replicaCount: 3
+  distribution-testerapi:
+    replicaCount: 3
+  distribution-testerweb:
+    replicaCount: 3
+  distribution-web:
+    replicaCount: 3
+license:
+  replicaCount: 3
+notification:
+  replicaCount: 3
+otp:
+  replicaCount: 3
+publish:
+  replicaCount: 3
+reporting:
+  replicaCount: 3
+resign:
+  replicaCount: 3
+resource:
+  replicaCount: 3
+schedulemanager:
+  replicaCount: 3
+signingidentity:
+  replicaCount: 3
+store:
+  store-web:
+    replicaCount: 3
+  store-admin:
+    replicaCount: 3
+  store-api:
+    replicaCount: 3
+  store-profile:
+    replicaCount: 3
+  store-report:
+    replicaCount: 3
+storesubmit:
+  replicaCount: 3
+taskserver:
+  replicaCount: 3
+web:
+  web-app:
+    replicaCount: 3
+  web-event:
+    replicaCount: 3
+webhook:
+  replicaCount: 3
+```
+
+## Values Table
+
+Users can view all available parameters of the Appcircle server Helm chart.
+
+### Parameters
+
+#### Global Parameters
+
+Example usage:
+
+```yaml
+# Example usage:
+global:
+  imageRegistry: "myregistry.com"
+```
+
+| Key                                                 | Type               | Default Value | Description                                 |
+| --------------------------------------------------- | ------------------ | ------------- | ------------------------------------------- |
+| global                                              | object             | -             | Application global configuration            |
```

Comment:
Table will have only customizable keys.
Remove empty keys.

Reply:
Asked @Alnyli07 for update since I can't be sure about which values can be configured or not.

Reply:
We will replace values yaml table later in next week. @burakberkkeskin